src/pages/gen2/start/index.mdx

```diff
@@ -0,0 +1,237 @@
+```
+
+Run your application with `npm run dev` and navigate to `http://localhost:3000`. You should now see the authenticator, which is already configured and ready for your first sign-up!
+
+## Data
+
+### Configure data resource
+
+```ts title="amplify/data/resource.ts"
+import { type ClientSchema, a, defineData } from '@aws-amplify/backend';
+
+const schema = a.schema({
+  Todo: a.model({
+    title: a.string(),
+    description: a.string(),
+    comment: a.string(),
+    priority: a.enum(['low', 'medium', 'high']).default('low')
+  })
+});
+
+export type Schema = ClientSchema<typeof schema>;
+
+export const data = defineData({
+  schema
+});
+```
+
+### Add data to your frontend
+
+Modify your app's home page file (e.g. `pages/index.tsx`) with the following contents:
+
+```ts title="pages/index.tsx"
+import { useState, useEffect } from 'react';
+import { generateClient } from 'aws-amplify/data';
+import { Schema } from '@/amplify/data/resource';
+
+// generate your data client using the Schema from your backend
+const client = generateClient<Schema>();
+
+export default function HomePage() {
+  const [todos, setTodos] = useState<Schema['Todo'][]>([]);
+
+  async function listTodos() {
+    const { data } = await client.models.Todo.list();
+    setTodos(data);
+  }
+
+  useEffect(() => {
+    listTodos();
+  }, []);
+
+  return (
+    <main>
+      <h1>Hello, Amplify 👋</h1>
+      <ul>
+        {todos.map((todo) => (
+          <li key={todo.id}>{todo.name}</li>
+        ))}
+      </ul>
+    </main>
+  );
+}
+```
+
+Once you save the file and navigate back to `http://localhost:3000` you should see an empty list of todos.
+
+## Iterating on your backend
+
+With Amplify running in [Sandbox mode](http://localhost:3000), try making a change to your data model and hit save. Changes are deployed in seconds.
+
+```ts {6} showLineNumbers
+import { type ClientSchema, a, defineData } from '@aws-amplify/backend';
+
+const schema = a.schema({
+  Todo: a.model({
+    title: a.string(),
+    description: a.string(),
+    comment: a.string(),
+    priority: a.enum(['low', 'medium', 'high']).default('low')
+  })
+});
+
+export type Schema = ClientSchema<typeof schema>;
+
+export const data = defineData({
+  schema
+});
+```
+
+## Deploy and host
+
+In order to share your app with others, Amplify will need to connect to a git repo as the source of truth for your project.
+
+### Create a remote git repository
+
+If you already have your project remotely hosted in a git repository, you can skip this step. Otherwise, navigate to your preferred git provider, and add your project to a new repo.
+
+Amplify supports the following git providers:
+
+- GitHub: `gh repo create`
+- CodeCatalyst: `something`
+- CodeCommit: `something`
+- Gitlab: `something`
+- Bitbucket: `something`
+
+### Connect repo to Amplify Console
+
+![d](/images/gen2/console1.jpg)
+
+### Manage your app
+
+![d](/images/gen2/console2.png)
+
+![d](/images/gen2/console3.png)
```

Comment:
Would a couple more sentences to call out what you want them to see on the images above. Also - are we going to add some recommended next steps at the end?

Reply:
@swaminator , thoughts on next steps here?